### DISCUSSION TITLE
split pdf visual editor / preview

### DISCUSSION THREAD

**Discussion by therealsatera** [Stirling-Tools/Stirling-PDF#576] [Ideas]
Hello everyone,

I knew from adobe online, there was a visual editor for splitting pdf documents. It's like you open in stirling the pdf in the multitool and you can set the split positions by clicking between pages. Then a line appeared, which you can drag / remove. After finishing the visual splitting, you can download the splitted files. 

Maybe it's possible with the multitool, but i can't find it.

So maybe someone can help me? Or count it as feature wish?

best regards 

therealsatera

**Comment by Frooodle** [comment]
Splitting via UI is not currently supported

**Comment by Frooodle** [comment]
Will add as feature request:)

**Reply by SephGER** [reply]
Are there any news on it?

**Reply by Frooodle** [reply]
It is added in our multi tool already!

**Reply by SephGER** [reply]
Sorry, totally forgot to up -d after the docker pull >_< tanks :D
